components/brave_rewards/resources/tip/components/tipSite.tsx

```diff
@@ -23,7 +24,17 @@ class TipSite extends React.Component<Props, {}> {
   }
 
   onTweet = () => {
-    this.actions.onTweet(this.props.publisher.name, '')
+    const url = this.props.url
+    if (!url || url.length === 0) {
+      return
+    }
+
+    let name = this.props.publisher.name
+    if (this.props.publisher.provider === 'twitter') {
+      name = `@${this.props.url.replace(/^.*\/(.*)$/, '$1')}`
```

Comment:
nit: `this.props.url` -> `url` as you defined it at top

Reply:
Ah yes, whoops - fixed.